
Adjust driver bits so it keeps the name of the replaced driver

Signed-off-by: Thomas Backlund <tmb@mageia.org>

---
 3rdparty/rtl8812au/Kconfig                       |    7 +++----
 3rdparty/rtl8812au/Makefile                      |    8 ++++----
 3rdparty/rtl8812au/hal/hal_com.c                 |    2 +-
 3rdparty/rtl8812au/os_dep/linux/ioctl_cfg80211.c |    2 +-
 4 files changed, 9 insertions(+), 10 deletions(-)

diff -Nurp linux-5.11/3rdparty/rtl8812au.orig/hal/hal_com.c linux-5.11/3rdparty/rtl8812au/hal/hal_com.c
--- linux-5.11/3rdparty/rtl8812au.orig/hal/hal_com.c	2021-03-07 18:43:20.313672664 +0200
+++ linux-5.11/3rdparty/rtl8812au/hal/hal_com.c	2021-03-07 18:43:51.392141299 +0200
@@ -12879,7 +12879,7 @@ bypass_hw_pg:
 	_rtw_memset(hal_data->EEPROMMACAddr, 0, ETH_ALEN);
 	ret = _FAIL;
 exit:
-	dev_info(&udev->dev, "88XXau %pM hw_info[%02x]", hw_addr, addr_offset);
+	dev_info(&udev->dev, "8812au %pM hw_info[%02x]", hw_addr, addr_offset);
 	return ret;
 }
 
diff -Nurp linux-5.11/3rdparty/rtl8812au.orig/Kconfig linux-5.11/3rdparty/rtl8812au/Kconfig
--- linux-5.11/3rdparty/rtl8812au.orig/Kconfig	2021-03-07 18:43:20.369675309 +0200
+++ linux-5.11/3rdparty/rtl8812au/Kconfig	2021-03-07 18:45:22.910465892 +0200
@@ -1,6 +1,5 @@
-config 88XXAU
-	tristate "Realtek 88XXau USB WiFi"
+config RTL8812AU
+	tristate "Realtek 8812au USB WiFi"
 	depends on USB
 	help
-	  Help message of 88XXau
-
+	  Help message of 8812au
diff -Nurp linux-5.11/3rdparty/rtl8812au.orig/Makefile linux-5.11/3rdparty/rtl8812au/Makefile
--- linux-5.11/3rdparty/rtl8812au.orig/Makefile	2021-03-07 18:43:20.370675357 +0200
+++ linux-5.11/3rdparty/rtl8812au/Makefile	2021-03-07 18:43:51.401141725 +0200
@@ -180,9 +180,9 @@ endif
 
 ifeq ($(CONFIG_RTL8812A)_$(CONFIG_RTL8821A)_$(CONFIG_RTL8814A), y_y_n)
 
-EXTRA_CFLAGS += -DDRV_NAME=\"rtl88XXau\"
+EXTRA_CFLAGS += -DDRV_NAME=\"rtl8812au\"
 ifeq ($(CONFIG_USB_HCI), y)
-USER_MODULE_NAME = 88XXau
+USER_MODULE_NAME = 8812au
 endif
 else
 EXTRA_CFLAGS += -DDRV_NAME=\"rtl8812au\"
@@ -2232,11 +2232,11 @@ $(MODULE_NAME)-y += $(_PLATFORM_FILES)
 
 $(MODULE_NAME)-$(CONFIG_MP_INCLUDED) += core/rtw_mp.o
 
-obj-$(CONFIG_88XXAU) := $(MODULE_NAME).o
+obj-$(CONFIG_RTL8812AU) := $(MODULE_NAME).o
 
 else
 
-export CONFIG_88XXAU = m
+export CONFIG_RTL8812AU = m
 
 all: modules
 
diff -Nurp linux-5.11/3rdparty/rtl8812au.orig/os_dep/linux/ioctl_cfg80211.c linux-5.11/3rdparty/rtl8812au/os_dep/linux/ioctl_cfg80211.c
--- linux-5.11/3rdparty/rtl8812au.orig/os_dep/linux/ioctl_cfg80211.c	2021-03-07 18:43:20.371675404 +0200
+++ linux-5.11/3rdparty/rtl8812au/os_dep/linux/ioctl_cfg80211.c	2021-03-07 18:43:51.402141772 +0200
@@ -2646,7 +2646,7 @@ void rtw_cfg80211_unlink_bss(_adapter *p
 #endif
 
 	if (!wiphy) {
-		pr_info("88XXAU: rtw_cfg80211_unlink_bss: wiphy is NULL\n");
+		pr_info("8812AU: rtw_cfg80211_unlink_bss: wiphy is NULL\n");
 		return;
 	}
 
